### DISCUSSION TITLE
Glitching on the first time opening a new file

### DISCUSSION THREAD

**Discussion by huyhoang8398** [nvim-tree/nvim-tree.lua#2356] [Q&A]
### Description

Nvimtree sidebar blinks file content of the first file open

### Neovim version

```text
NVIM v0.9.1
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

macOS 11.7.6

### nvim-tree version

0a89dcb

### Clean room replication

```lua
I dont really know which plugin cause this
```

### Steps to reproduce

1. NvimTreeOpen
2. Telescope find_files
3. Open that file

### Expected behavior

No code blinking in nvimtree sidebar

### Actual behavior

https://github.com/nvim-tree/nvim-tree.lua/assets/25379882/d27bd7e3-128f-4fae-84b7-1e0d57c34b28

**Comment by alex-courtis** [comment]
This is working as intended:

Telescope is opening the file in the nvim-tree buffer. nvim-tree ejects it, which is the default behaviour. See `:help nvim-tree.actions.open_file.eject`

My solution is to focus the window in which I want to open the buffer _before_ opening telescope. I call this "go home" and my mappings invoke it before many actions.

Warning, complexity ahead: https://github.com/alex-courtis/arch/blob/3ab6eab8598a707349f9eb6415db569bb2e883fc/config/nvim/lua/amc/windows.lua#L27
